docs/doxygen/pages.dox

```diff
@@ -16,22 +16,25 @@ The configurations for memory estimation are defined <a href="https://github.com
 
 <table>
     <tr>
-        <td colspan="3"><b>Code Size of coreJSON (example generated with GCC for ARM Cortex-M)</b></td>
+        <td colspan="4"><center><b>Code Size of coreJSON library files (sizes generated with [GCC for ARM Cortex-M toolchain 20191025](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads/9-2019-q4-major))</b></center></td>
```

Comment:
> 20191025

Is there any specific reason for this? I would've thought we'd be using the latest version of compilers

Reply:
It's just the version I used but also the same one used in v4_beta.